torchsde/_core/sdeint.py

```diff
@@ -21,22 +21,24 @@
 from . import misc
 from .._brownian import BaseBrownian, BrownianInterval
 from ..settings import LEVY_AREA_APPROXIMATIONS, METHODS, NOISE_TYPES, SDE_TYPES
-from ..types import Any, Dict, Optional, Scalar, Tensor, TensorOrTensors, Vector
+from ..types import Any, Dict, Optional, Scalar, Tensor, TensorOrTensors, Tuple, Vector
 
 
 def sdeint(sde,
            y0: Tensor,
            ts: Vector,
            bm: Optional[BaseBrownian] = None,
            method: Optional[str] = None,
-           dt: Optional[Scalar] = 1e-3,
-           adaptive: Optional[bool] = False,
-           rtol: Optional[Scalar] = 1e-5,
-           atol: Optional[Scalar] = 1e-4,
-           dt_min: Optional[Scalar] = 1e-5,
+           dt: Scalar = 1e-3,
+           adaptive: bool = False,
+           rtol: Scalar = 1e-5,
+           atol: Scalar = 1e-4,
+           dt_min: Scalar = 1e-5,
            options: Optional[Dict[str, Any]] = None,
            names: Optional[Dict[str, str]] = None,
-           logqp: Optional[bool] = False,
+           logqp: bool = False,
+           extra: bool = False,
+           extra_solver_state: Optional[Tuple[Tensor, ...]] = None,
```

Comment:
same as above, `TensorOrTensors` might be better

Reply:
It's never a single tensor, however. Even for a future hypothetical case where we only need a single extra solver state, I'd rather leave it as a 1-tuple of a tensor, for consistency. (Not really a strong feeling though.)

Reply:
Right, but even in the tuple of tensors case, we have the more encompassing `Tensors` type.

Reply:
Had missed that we had `Tensors`. agreed.